app/controllers/concerns/conferencable.rb

```diff
@@ -0,0 +1,18 @@
+module Conferencable
+    extend ActiveSupport::Concern
+    
+    def index
+      @conferences = conferences
+    end
+
+    def conferences
+        Conference.ordered(sort_params).in_current_season
```

Comment:
I know this PR just moves existing code into a different file, but while we're at it: what do you think about removing this method entirely and call it in `index` directly? It's not complex logic and I would argue that it improves the narrative of the code to remove that level of indirection.